python/cugraph/cugraph/dask/centrality/katz_centrality.py

```diff
@@ -99,6 +103,7 @@ def katz_centrality(input_graph,
         Setting too small a tolerance can lead to non-convergence due to
         numerical roundoff. Usually values between 1e-2 and 1e-6 are
         acceptable.
+
```

Comment:
This line is fine but there should also be a blank line above `normalized` below.